server/protocols/jmap/src/main/java/org/apache/james/jmap/jwt/JWTAuthenticationStrategy.java

```diff
@@ -37,14 +38,15 @@
 
 public class JWTAuthenticationStrategy implements AuthenticationStrategy {
 
-    @VisibleForTesting static final String AUTHORIZATION_HEADER_PREFIX = "Bearer ";
+    @VisibleForTesting
+    public static final String AUTHORIZATION_HEADER_PREFIX = "Bearer ";
     private final JwtTokenVerifier tokenManager;
     private final MailboxManager mailboxManager;
     private final UsersRepository usersRepository;
 
     @Inject
     @VisibleForTesting
-    JWTAuthenticationStrategy(JwtTokenVerifier tokenManager, MailboxManager mailboxManager, UsersRepository usersRepository) {
+    public JWTAuthenticationStrategy(JwtTokenVerifier tokenManager, MailboxManager mailboxManager, UsersRepository usersRepository) {
```

Comment:
Why does this needs to be public?

Reply:
Why still?

Reply:
Because this class and it's test are different packages.